components/Workspaces/WorkspaceLayout.tsx

```diff
@@ -50,7 +50,7 @@ export const WorkspaceLayout = ({ workspaceId, banner, children, footer }: Works
           <TopNav />
           <ClientOnly>
             <span ref={sidebarRef}>
-              <AppSideBar workspaceId={workspaceId} hideSidebar={hideSidebar} sidebarCollapsed={showingSidebar} />
+              <AppSideBar workspaceId={workspaceId} hideSidebar={hideSidebar} sidebarCollapsed={!showingSidebar} />
```

Comment:
This should have always been `!showingSidebar` to set the `sidebarCollapsed` prop properly.